app/assets/src/components/views/phylo_tree/PhyloTreeCreation.jsx

```diff
@@ -13,7 +13,7 @@ class PhyloTreeCreation extends React.Component {
   constructor(props) {
     super(props);
 
-    this.minNumberOfSamples = 4;
+    this.minNumberOfSamples = 2;
```

Comment:
Can you add some comments to the limits explaining the rationale for the choices.
In this case, do we complement with NCBI references (since a tree with only two branches might not make much sense)?

Reply:
You're right, 4 data points are needed for a tree. However, it seems people who only have a couple samples uploaded get confused when they can't make a tree. I chose 2 because typically we pull in 1 NCBI reference per sample, so the total would come to 4. I haven't completely decided whether to go through with this change, because the point you make is valid and users SHOULD recognize that...

The correct solution in the future will be to add an explicit NCBI reference selector in the workflow (which we postponed to later), and reinstate the limit of 4.

Reply:
Update: added comment for one of the limits, removed the change to the 4 threshold for now